### PR TITLE
Allow API GW access token override

### PR DESCRIPTION
**Pull request by mdeuser**
- Can change API GW authentication from basic auth to another `Authorization` header value.

### CODE REVIEW

File: commands/api.go [hunk 1]
```diff
@@ -1010,15 +1013,22 @@ func getAccessToken() (string, error) {
 	var token string = "DUMMY TOKEN"
 	var err error
 
-	props, err := ReadProps(Properties.PropsFile)
-	if err == nil {
-		if len(props["APIGW_ACCESS_TOKEN"]) > 0 {
-			token = props["APIGW_ACCESS_TOKEN"]
-		}
+	// If the api gw access token override has been set, use it instead of the default
+	whisk.Debug(whisk.DbgInfo, "api gw access token override is '%s'\n", ApiGwAccessToken)
```
Comment (dubee) [inline commands/api.go]: @mdeuser, might not want to print the access token if it is sensitive information.

Comment (mdeuser) [reply commands/api.go]: i just checked the `bx` cli, and id does not display even these "short lived" tokens..

**Comment by mdeuser** [comment]
bumping travis

**Comment by mdeuser** [comment]
api gw tests all pass.. but travis is failing here..
```
stderr: error: Unable to obtain API build information: Get http://172.17.0.1:10001/api/v1: read tcp 172.17.0.1:47200->172.17.0.1:10001: read: connection reset by peer
```
trying travis again

**Comment by mdeuser** [comment]
fixed that travis failure.. i think a recent ssl change may have caused that test to fail.  all good now @dubee

**Comment by houshengbo** [comment]
@mdeuser Could you rebase this PR on the latest code?

**Comment by mdeuser** [comment]
@houshengbo - actually, this pr was superseded by pr #381, so i'm closing this pr which is no longer needed.

**Pull request closed without merging**